ansible/roles/linux-ansible-init/files/ansible-init.service

```diff
@@ -5,10 +5,10 @@ After=network-online.target
 ConditionPathExists=!/var/lib/ansible-init.done
 
 [Service]
-Type=oneshot
+Type=simple
```

Comment:
Can we make this `oneshot` on >= 20.04, since that is actually the correct type, and only apply this change for 18.04?

See the `ansible_distribution*` variables here - https://docs.ansible.com/ansible/latest/user_guide/playbooks_vars_facts.html

Reply:
To do that neatly, I'd suggest moving the ansible-init.service file to ansible-init.service.j2, putting it in the templates file and then making adding the conditional:
```
{% if ansible_distribution_version == 18.04 %}
Type=simple
{% else %}
Type=oneshot
{% endif %}
```

Reply:
Sounds reasonable

Reply:
Removed that statement in commit `46328c6`